compiler/modules/front.syntax/parseTypeDesc.bal

```diff
@@ -233,6 +233,20 @@ function parseFunctionTypeDesc(Tokenizer tok, FunctionParam[]? namedParams = ())
                 params.push(param);
                 check tok.advance();
             }
+            "..." => {
+                td = { member: td, startPos: td.startPos, endPos: tok.currentEndPos(), dimensions: [()] }; // convert to array type desc
```

Comment:
Conversion to array type should be in resolveTypes. Syntax tree for a parameter of `int...` should have type as `int` and isRest true.

Reply:
fixed